FaceRecognizer.py

```diff
@@ -89,10 +91,15 @@ def shape_to_np(shape, dtype="int"):
     (x, y, w, h) = face_utils.rect_to_bb(rect)
     faceOrig = imutils.resize(image[y:y + h, x:x + w], width=256)
     faceAligned = face.align(image, gray, rect)
-    cv2.rectangle(image, (x, y), (x + w, y + h), (0, 255, 0), 2)
 
+    gray= cv2.cvtColor(faceAligned, cv2.COLOR_BGR2GRAY)
+    area=cv2.boundingRect(gray)
+    faceAligned = faceAligned[area[1]:area[1] + area[3], area[0]:area[0] + area[2]]
+

```

Comment:
cv2.boundingRect() method found maximum minimum coordinates of the colored part.
and Leave only the colored part.
area[1]=> min row       area[0]=>min col
area[3]=> max row      area[2]=>max col